src/App.js

```diff
@@ -6,15 +6,17 @@ import HackDetails from "./components/hack-details";
 import Login from "./components/Login";
 import Register from "./components/register";
 import Header from "./components/header";
+import Login from "./components/Login"
+import Home from "./components/Home";
 import "./css/main.css";
 
 function App() {
   return (
     <BrowserRouter>
       <Header />
       <Switch>
-        {/* <Route exact path="/" component={<h1>This is root</h1>}/> */}
-        <Route path="/login" component={Login} />
+        <Route exact path="/" component={Home} />
+        <Route exact path="/login" component={Login} />
```

Comment:
do you need the exact on this?

Reply:
No we do not, I think I changed the original home route to Login and forgot to remove the exact!